WooCommerce/Classes/ViewRelated/Products/Edit Product/ProductFormViewController.swift

```diff
@@ -72,10 +72,10 @@ final class ProductFormViewController<ViewModel: ProductFormViewModelProtocol>:
     private let aiEligibilityChecker: ProductFormAIEligibilityChecker
     private var descriptionAICoordinator: ProductDescriptionAICoordinator?
 
-    private lazy var tooltipUseCase = ProductDescriptionAITooltipUseCase()
+    private var tooltipUseCase: ProductDescriptionAITooltipUseCase?
```

Comment:
❓ any reasons for changing the lazy var to an optional variable? I thought it was nice initializing it only when needed and it's never reset

Reply:
Good point. Done in https://github.com/woocommerce/woocommerce-ios/pull/10073/commits/13cfd2a6d7f10b998af8455bb6f7677a2c4f5ea6